file/compressors.cc

```diff
@@ -5,7 +5,7 @@
 #include "file/compressors.h"
 
 #include <zlib.h>
-#include <lz4.h>
+#include "lz4.h"
```

Comment:
why?

Reply:
https://docs.bazel.build/versions/master/bazel-and-cpp.html
> Use quoted includes (#include "foo/bar/baz.h") for non-system headers, not angle-brackets (#include <foo/bar/baz.h>).

Reply:
Its a system header in a sense. Third party library is like a system headerin this context.

Reply:
WDYM? this doesn't work in bazel, does cmake treat third party deps as system?

Reply:
May target_include_directories(SYSTEM) solve that issue?

> The trick is to use target_include_directories(my_library SYSTEM PUBLIC include/). The SYSTEM turns the include directory into a system include directory. Compilers will not issue warnings from header files originating from there. https://foonathan.net/2018/10/cmake-warnings/

Reply:
@blockspacer It will work fine with cmake, but this PR is to add Bazel support.
I will check out if it's possible to do it with bazel as well, but I do wonder thought why treat 3party as System?

Reply:
@romange Is this a blocker for this PR?

Reply:
Yes, @zacharya19, look at the third_party option they mention. You should not change the source code in this PR